config/routes.rb

```diff
@@ -32,12 +32,6 @@
     post '/api/usps_upload' => 'gpo_upload#create'
   end
 
-  scope module: :lambda_callback do
-    post '/api/proofing_results/address/:result_id' => 'address_proof_result#create', as: 'address_proof_result'
-    post '/api/proofing_results/resolution/:result_id' => 'resolution_proof_result#create', as: 'resolution_proof_result'
-    post '/api/proofing_results/document/:result_id' => 'document_proof_result#create', as: 'document_proof_result'
-  end
```

Comment:
If this was actively deployed, I would remove these endpoints in a separate URL

However, we're transition to ruby workers already in the environments we care about (dev, int) so I figured I'd remove these while I remembered